vllm/model_executor/layers/quantization/base_config.py

```diff
@@ -5,8 +5,9 @@
 from torch import nn
 
 
-class QuantizeMethodBase(ABC):
-    """Base class for different quantized methods."""
+class QuantizableMethodBase(ABC):
+    """Base class for different methods that may be quantized."""
+    QUANTIZED = False
```

Comment:
@robertgshaw2-neuralmagic 

Renamed `QuantizeMethodBase` to more accurate `QuantizableMethodBase` since non-quantized methods also inherit this.